charms/jupyter-ui/tests/unit/test_operator.py

```diff
@@ -147,59 +291,227 @@ def test_deploy_k8s_resources_success(
         assert isinstance(harness.charm.model.unit.status, MaintenanceStatus)
 
     @pytest.mark.parametrize(
-        "config_key,expected_images",
+        "config_key,expected_images_yaml",
```

Comment:
I guess this is not just images.

Reply:
Good catch!  Renamed these (both here and in the function)